### PR TITLE
Organize db session

### PR DESCRIPTION
**Pull request by maxcnunes**
Moves db’s main process code to a proper file. And makes actions not aware of main process.

### CODE REVIEW

File: src/actions/databases.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const getDB = require('remote').require('./src/db').getDB;
+import { getDBSession } from '../db';
```
Comment (krolow) [inline src/actions/databases.js]: about the name, "session" looks a bit strange for me... perhaps we can use getDBConnection or some other thing, it's more a database connection or connection instance than a session...

what do you think?

Comment (maxcnunes) [inline src/actions/databases.js]: I didn't like the name either. I had tought in name it `getDBConnection` as well but actually it is more than only a database connection. It is the sqlectron-db that is a wrap over knex.js. In the end I chose Session inspired on [nhinbernate](http://nhibernate.info/doc/nh/en/index.html#quickstart-playingwithcats).

@esnunes what do you think?

Comment (esnunes) [inline src/actions/databases.js]: I think the concept of a `Server Session` fits to the requirements due to the fact that the *back-end* of the application must handle tunnels and connections issues like a drop of a connection.

Comment (maxcnunes) [inline src/actions/databases.js]: So is it gonna be `getServerSession`?

Comment (maxcnunes) [inline src/actions/databases.js]: Merging it to master. We can pick up a better name later.

**Pull request merged**